app/controllers/iiif_token_controller.rb

```diff
@@ -8,7 +8,7 @@ def create
     @message = if token
                  {
                    accessToken: token,
-                   tokenType: 'Token',
+                   tokenType: 'Bearer',
```

Comment:
I think `tokenType` is gone in IIIF Auth 1.0; I'm not sure if we had backwards compatibility concerns that led us to keep it as-is?

Reply:
Yeah I hadn't realized that. /shrug should we remove it?